ament_cmake_test/ament_cmake_test/__init__.py

```diff
@@ -304,17 +304,18 @@ def _generate_result(result_file, *, failure_message=None, skip=False):
         '<skipped type="skip" message="">![CDATA[Test Skipped by developer]]</skipped>' \
         if skip else ''
     return """<?xml version="1.0" encoding="UTF-8"?>
-<testsuite name="%s" tests="1" failures="%d" time="0" errors="0" skipped="%d">
+<testsuite name="%s" tests="1" failures="%d" time="0" errors="%d" skipped="%d">
   <testcase classname="%s" name="%s.missing_result" time="0">
-    %s%s
+    %s%s%s
```

Comment:
Should the error message be added here literally - without any tag around it?

Reply:
uh, no. Fixed in b1d0eae401565f81390ab82963fdc3415d9b6215. I'm going to launch a test run.